tools/python/m2k-osc/m2k-osc.py

```diff
@@ -0,0 +1,69 @@
+import matplotlib
+import numpy as np
+import tkinter as tk
+from tkinter import ttk
+import matplotlib.figure as  fg
+import libm2k
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib import style
+import threading
+import time
+from ctx_tab import *
+from ain_tab import *
+from aout_tab import *
+from dig_tab import *
+from ps_tab import *
+
+LARGE_FONT = ("Verdana",12)
+SMALL_FONT = ("Vedana",10)
+
+#style = ttk.Style()
+#style.theme_create("MyStyle",parent="alt",settings={"TNotebook":{"configure":{"tabmargins":[2,5,2,0]}},"TNotebook.Tab":{"configure":{"padding":[100,100]},}})
+#style.theme_use("MyStyle")
+
+class m2k_osc_app(tk.Frame):
+    def __init__(self,parent,*args,**kwargs):
+        tk.Frame.__init__(self,parent,*args,**kwargs)
+        tab_parent = ttk.Notebook()
+        self.tab_parent = tab_parent
+        tab_ctx = ctx_tab_frame(tab_parent, self)
+        tab_parent.add(tab_ctx,text='Context' )
+
+        tab_parent.grid(row=0,column=0)
+        tab_parent.grid_columnconfigure(0,weight=1)
+        tab_parent.grid_rowconfigure(0,weight=1)
+
+        """
```

Comment:
for the final version, it's a good idea to remove commented code;
since this is a draft, it's not a problem;

Reply:
commented sections removed